From bb33cdde8d36fcc3d10b2ebd1001b18347755a07 Mon Sep 17 00:00:00 2001
From: Li Zhou <li.zhou@windriver.com>
Date: Thu, 28 Apr 2022 10:16:20 +0800
Subject: [PATCH 5/5] kernel-rt: add '-rt' to abiname

Adding an extra '-rt' to abiname can change '6.6.0-1-amd64'
in file names/paths to '6.6.0-1-rt-amd64'.
In this way, the same names/paths in kernel-rt and kernel-std
packages can be avoided.

Signed-off-by: Li Zhou <li.zhou@windriver.com>
[lz: Adapted the patch for upgrading kernel from 5.10 to 6.6]
Signed-off-by: Li Zhou <li.zhou@windriver.com>
---
 debian/config/defines | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/debian/config/defines b/debian/config/defines
index 37ee655..f1ca81b 100644
--- a/debian/config/defines
+++ b/debian/config/defines
@@ -1,5 +1,5 @@
 [abi]
-abiname: 1
+abiname: 1-rt
 ignore-changes:
  __cpuhp_*
  __udp_gso_segment
-- 
2.17.1

